spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -32,6 +32,12 @@ private SparkReadOptions() {}
   // End snapshot ID used in incremental scans (inclusive)
   public static final String END_SNAPSHOT_ID = "end-snapshot-id";
 
+  // Start timestamp used in multi-snapshot scans (exclusive)
+  public static final String START_TIMESTAMP = "start-timestamp";
```

Comment:
Looks like Flink has a config called 'start-snapshot-timestamp'.  Its a bit wordy but wondering if we need to synchronize the two (if they are related):  cc @stevenzwu @pvary @hililiwei

Reply:
Yes, it's called "start-snapshot-timestamp" in flink.

https://github.com/apache/iceberg/blob/6d47097151b4df1d3269563a7ebfdb4b6c270a64/flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkReadOptions.java#L48-L49

Reply:
Thanks @szehon-ho and @hililiwei for the input. It'd be nice to consolidate. I kind of prefer the name `start-timestamp` here, so that users don't have to worry about the concept `snapshot`, the query is purely based on the time range.